doc/contributing/technical-priorities.md

```diff
@@ -161,17 +161,16 @@ As TypeScript usage continues to grow and gains more prominence in the
 ecosystem, enhancing its support is essential for delivering an improved
 developer experience for newcomers and experienced users alike.
 
-## Non-goals
-
-The following are not considered technical priorities for the project:
-
-* **Unbundling `npm`**. When `npm` was included in the Node.js distribution, it
-  was the only JavaScript package manager available, and it was provided as a
-  way to help developers easily install other JavaScript software. `npm` is also
-  the reference implementation for the npm registry, which is the de facto
-  primary source for most JavaScript software. Today, `npm` is one of many
-  high-quality options. However, the potential removal of `npm` would be a very
-  disruptive breaking change, even as a semver-major change, and therefore it is
-  not a goal of the Node.js project to work toward such a goal. The `npm`
-  library included with the Node.js distribution can be freely used as a
-  dependency of other parts of the Node.js distribution.
+## Package management
+
+The ability to easily install and manage dependencies and development tools is a
+key part of the developer experience, and therefore Node.js must provide a
```

Comment:
Wouldn't "developer experience" refer to the experience of maintainers when we mean "users of Node.js (who are developers)"?
```suggestion
key part of the user experience, and for that reason Node.js must provide a
```

Reply:
I consider them interchangeable, but sure, either is fine with me.